lib/helpers/variables_interface.rb

```diff
@@ -54,10 +54,10 @@ def display_items_of_class(item, index)
       general_item = "[#{index}] - Class: #{item.class} Id: #{item.id}, Published at: #{item.published_date} "
       book_chr = "Characteristics: #{item.publisher}, #{item.cover_state}"
       general_item + book_chr
-    when 'Game'
-      general_item = "[#{index}] - Class: #{item.class} Id: #{item.id}, Published at: #{item.published_date} "
-      game_chr = "Characteristics: #{item.multiplayer}, #{item.last_played_at}"
-      general_item + game_chr
+    # when 'Game'
+    #   general_item = "[#{index}] - Class: #{item.class} Id: #{item.id}, Published at: #{item.published_date} "
+    #   game_chr = "Characteristics: #{item.multiplayer}, #{item.last_played_at}"
+    #   general_item + game_chr
```

Comment:
I suggest leaving that part uncommented even if you are not going to implement an add_author option in the CLI app this part would actually do something for my code. A Label class should accept all classes that inherit from the Item class.

Reply:
Yes, sorry I really was checking if this code maybe do something in game_module and forgot to uncommented again 😅